stable/kubernetes-dashboard/values.yaml

```diff
@@ -188,12 +164,11 @@ podDisruptionBudget:
   minAvailable:
   maxUnavailable:
 
-
 ## PodSecurityContext for pod level securityContext
-##
-# securityContext:
-#   runAsUser: 1001
-#   runAsGroup: 2001
-securityContext: {}
+securityContext:
+  allowPrivilegeEscalation: false
+  readOnlyRootFilesystem: true
```

Comment:
I just ran this chart in helm 3 / kubernetes 1.17.0 and got this error:

```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Deployment.spec.template.spec.securityContext): unknown field "allowPrivilegeEscalation" in io.k8s.api.core.v1.PodSecurityContext, ValidationError(Deployment.spec.template.spec.securityContext): unknown field "readOnlyRootFilesystem" in io.k8s.api.core.v1.PodSecurityContext]
```

Something does not like `allowPrivilegeEscalation` and `readOnlyRootFilesystem` 🤔
Manual removal of both fields solved the issue, I'm running `v2.0.0-beta8+0.g4275399` now

Reply:
Thanks! Indeed, the PodSecurityContext and SecurityContext was mixed. Fixed at https://github.com/kubernetes/dashboard/pull/4502.